app/services/tools/chdman.py

```
@@ -149,6 +162,53 @@ def convert(
+            title = disc_info.get("title") or game_id
+            embedded = await embed_in_chd(output_path, game_id, title, self.binary_path)
+            if embedded:
+                logger.info(
+                    "Embedded disc ID %r in %s", game_id, Path(output_path).name,
+                )
+            else:
+                logger.debug(
+                    "Failed to embed disc ID %r in %s",
+                    game_id, Path(output_path).name,
+                )
+        except Exception as exc:  # best effort; tagging never fails the job
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)